reactor-core/src/main/java/reactor/core/publisher/VoidProcessor.java

```diff
@@ -16,7 +16,7 @@
 import reactor.util.annotation.Nullable;
 import reactor.util.context.Context;
 
-final class VoidProcessor<T> extends MonoProcessor<T> implements Sinks.One<T> {
+final class VoidProcessor<T> extends MonoProcessor<T> implements InternalOneSink<T> {
```

Comment:
I guess there's no real reason for `VoidProcessor` to implement `One` rather than `Empty`, other than the fact that it was originally based on existing `MonoProcessor`... Follow-up issue maybe?

Reply:
https://github.com/reactor/reactor-core/issues/2408